### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by LightningDev23**
Just fixed up a few grammar things. Let me know if you have suggestions or disagree with any of my changes.

### CODE REVIEW

File: src/components/NavBars/DmContent/Friends/BlockedFriends.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { ChevronDown, EllipsisVertical } from "lucide-react";
 const columns = [
 	{ name: "ID", uid: "id", sortable: true },
 	{ name: "USERNAME", uid: "username", sortable: false },
-	{ name: "Blocked Since", uid: "blockedSince", sortable: true },
+	{ name: "BLOCKED SINCE", uid: "blockedSince", sortable: true },
```
Comment (Kodarru) [inline src/components/NavBars/DmContent/Friends/BlockedFriends.tsx:23]: Capitalizing `Blocked Since` makes no sense to me, up to Darker

**Review by Kodarru — CHANGES_REQUESTED** [review]

**Comment by Kodarru** [comment]
Also overall change needed: new-lines on files

**Comment by Kodarru** [comment]
Other than that, LGTM 👍

File: src/components/NavBars/DmContent/Friends/BlockedFriends.tsx (hunk 1 above)
Comment (Darker-Ink) [reply src/components/NavBars/DmContent/Friends/BlockedFriends.tsx:23]: this matches everything else, so its fine, in the future it may be changed though

**Review by Darker-Ink — APPROVED** [review]
For now this will be accepted, for future reference this stuff will be moved to `public/locales/en.json` I just cba right now

**Pull request merged**